public/index.html

```diff
@@ -31,6 +31,23 @@
         apiEndpoint: "http://127.0.0.1:7777",
       };
     </script>
+    <script type="text/javascript">
+      window.smartlook ||
+        (function (d) {
+          var o = (smartlook = function () {
+              o.api.push(arguments);
+            }),
+            h = d.getElementsByTagName("head")[0];
+          var c = d.createElement("script");
+          o.api = new Array();
+          c.async = true;
+          c.type = "text/javascript";
+          c.charset = "utf-8";
+          c.src = "https://rec.smartlook.com/recorder.js";
+          h.appendChild(c);
+        })(document);
+      smartlook("init", "1b5b728afd698b6544fbbb289a1c0071bafddca0");
```

Comment:
Please tell me this is not our API key. 😂

Reply:
It's our project id - which seems OK to have public. We haven't generated private tokens to access their api.
![Cursor_and_Projects___Smartlook](https://user-images.githubusercontent.com/1753324/103587733-c14f5a80-4e9c-11eb-8205-04ebca0b05cd.jpg)